routers/router.js

```diff
@@ -7,5 +7,6 @@ router.get('/', TaskController.findAllTasks)
 router.post('/', TaskController.addTask)
 router.put('/:id', TaskController.updateTask)
 router.delete('/:id', TaskController.deleteTask)
+router.delete('/', TaskController.deleteCompleted)
```

Comment:
Remove this route. As I told you above, we don't need to define another function to delete only completed tasks. Please handle it on the front-end side. Besides that, the way you define a delete method to the '/' path is very dangerous. We never do this.

Reply:
I'm sorry about this, I fixed this